src/FramedClient.ts

```diff
@@ -125,10 +127,11 @@ class FramedClient {
         })
     }
 
-    public async getViewStatus(fileMetadata: FileMetadata, chatToken: OmnichannelChatToken | null = null): Promise<void> {
+    public async getViewStatus(fileMetadata: FileMetadata, chatToken: OmnichannelChatToken | null = null, supportedImagesMimeTypes?: string[]): Promise<void> {

```

Comment:
Please add `[]` as default value for `supportedImagesMimeTypes` for backward compatibility.

Reply:
done